openapi.yaml

```diff
@@ -38,9 +38,10 @@ paths:
         '200':
           description: Template successfully generated.
           content:
-            application/json:
+            application/octet-stream:
```

Comment:
what is the difference between `application/octet-stream` and `application/zip`? I'm asking because in my tests we return the `application/zip` Content-Type:
![imagem](https://user-images.githubusercontent.com/18630253/149823958-94144db1-e88f-41f6-b7e6-a437e3199d58.png)

I believe we are sending `zip` because of this line: https://github.com/asyncapi/server-api/blob/03c5c53a351e54d07e5ff767ad25ee8295f6a800/src/services/archiver.service.ts#L19

Since the filename argument has a `.zip` extension, the header is set to that. But should we send just a binary file without specifying it's a zip to be more generic, what does everyone think?

Reply:
Thats a really good catch @BOLT04 ! Then `application/zip` is more accurate. `application/octet-stream` would be valid as well to as `application/zip` inherits the features from `application/octet-stream` but it's not accurate.

As per IANA, it is an accepted MIME type since 1993 https://www.iana.org/assignments/media-types/application/zip

Reply:
Ok, changed to the `application/zip`. Thanks @BOLT04 for that catch! :)